src/pages/get-involved.js

```diff
@@ -0,0 +1,222 @@
+import React from "react";
+import { animateScroll as scroll } from 'react-scroll';
+import Layout from "../components/common/layout";
+import SEO from "../components/common/seo";
+import Button from "../components/common/button";
+import { BreakPoint, mq } from "../util/mq";
+import { css } from "@emotion/core";
+import HeroBackgroundImage from "../components/operational-review/hero-background-image";
+import linkedInImage from "../images/LinkedIn-Logos/LI-In-Bug.png";
```

Comment:
Can these paths be lowercase only? I don't want to deal with case-sensitive filepath-clash bugs, and it just looks weird.